spec/unit/postmark/account_api_client_spec.rb

```diff
@@ -728,5 +728,46 @@
       end
     end
 
+    describe '#get_data_removal_status' do
+      let(:response) {
+        {
+          "ID" => 42,
+          "Status" => "Done"
+        }
+      }
+
+      it 'performs a GET request to /data-removals/:id endpoint' do
+        allow(subject.http_client).to receive(:get).
+          with('data-removals/42').and_return(response)
+        subject.get_data_removal_status(42)
+      end
+
+      it 'formats the keys of returned response' do
+        allow(subject.http_client).to receive(:get).and_return(response)
+        keys = subject.get_data_removal_status(42).keys
+        expect(keys.all? { |k| k.is_a?(Symbol) }).to be true
+      end
+    end
+
+    describe '#request_data_removal' do
+      let(:response) {
+        {
+          "ID" => 42,
+          "Status" => "Done"
+        }
+      }
+
+      it 'performs a POST request to /data-removals endpoint' do
+        allow(subject.http_client).to receive(:post).
+          with('data-removals', an_instance_of(String)).and_return(response)
+        subject.request_data_removal(:foo => 'bar')
```

Comment:
wondering if we should use here more descriptive example data, like requested_by: .. etc. I know it's not mandatory here, but would make the test bit more descriptive, with having non random data for requesting removal.

Reply:
@ibalosh Yeh, this felt a little awkward to me too, but it seemed to be the done thing in this project, ie: 

https://github.com/ActiveCampaign/postmark-gem/blob/f4938b4d89629ead588485c54ee1623d5962a3a6/spec/unit/postmark/account_api_client_spec.rb#L624

I think these tests could do with a refresher at some point, but will stick with the local convention for now!